### PR TITLE
Harshil/ref 5677

### PR DESCRIPTION
**Pull request by harshilpm**
big reorg.
nuking com.parallelmachines.reflex
& tree will be as following now

[reflex_algo_tree.log](https://github.com/mlpiper/mlpiper/files/2954264/reflex_algo_tree.log)

### CODE REVIEW

File: reflex-algos/src/main/python/deputy/build/lib/parallelm/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (yakov-g) [inline reflex-algos/src/main/python/deputy/build/lib/parallelm/__init__.py]: build folder should not be here!!!!

Comment (yakov-g) [reply reflex-algos/src/main/python/deputy/build/lib/parallelm/__init__.py]: Deputy should not be here at all!

**Review by yakov-g — CHANGES_REQUESTED** [review]
Clean up deputy folder

**Comment by harshilpm** [comment]
[reflex_algo_tree.log](https://github.com/mlpiper/mlpiper/files/2954515/reflex_algo_tree.log)

**Pull request merged**